codec-http2/src/main/java/io/netty/handler/codec/http2/internal/hpack/HuffmanDecoder.java

```diff
@@ -213,4 +137,96 @@ private static void insert(Node root, int symbol, int code, byte length) {
             current.children[i] = terminal;
         }
     }
+
+    private static final class DecoderProcessor implements ByteProcessor {
+        private final ByteArrayOutputStream baos = new ByteArrayOutputStream();
```

Comment:
was the plan to implement a `AppendableByteSequence` (just like `AppendableCharSequence`)?